### PR TITLE
Add Blorc.OpenIdConnect

### PR DESCRIPTION
**Pull request by alexfdezsauco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Note: the Blazor Visual Studio extension is no longer required since .NET Core 3
...
 * [Blazor.OpenId](https://github.com/jbomhold3/Blazor.Openid) - ![stars](https://img.shields.io/github/stars/jbomhold3/Blazor.Openid?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/jbomhold3/Blazor.Openid?style=flat-square&cacheSeconds=86400) Easy authentication and client refresh using OpenId.
+* [Blorc.OpenIdConnect](https://github.com/WildGums/Blorc.OpenIdConnect) - ![stars](https://img.shields.io/github/stars/WildGums/Blorc.OpenIdConnect?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/WildGums/Blorc.OpenIdConnect?style=flat-square&cacheSeconds=86400) The right way to use OpenIdConnect on Blazor.
```
Comment (danroth27) [inline README.md:71]:
```suggestion
* [Blorc.OpenIdConnect](https://github.com/WildGums/Blorc.OpenIdConnect) - ![stars](https://img.shields.io/github/stars/WildGums/Blorc.OpenIdConnect?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/WildGums/Blorc.OpenIdConnect?style=flat-square&cacheSeconds=86400) The right way to use OpenID Connect on Blazor.
```

File: README.md [hunk 2]
```diff
@@ -210,7 +211,7 @@ Note: the Blazor Visual Studio extension is no longer required since .NET Core 3
...
 * [Blazor.SpinKit](https://github.com/faso/Faso.Blazor.SpinKit) - ![last commit](https://img.shields.io/github/last-commit/faso/Faso.Blazor.SpinKit?style=flat-square&cacheSeconds=86400) Blazor components for [SpinKit](https://tobiasahlin.com/spinkit/) spinners.
-* [Blorc.PatternFly](https://github.com/WildGums/Blorc.PatternFly) - ![last commit](https://img.shields.io/github/last-commit/WildGums/Blorc.PatternFly?style=flat-square&cacheSeconds=86400) Blazor wrappers for [PatternFly](https://www.patternfly.org). To view the latest develop branch in action, visit the [demo](http://blorc-patternfly.wildgums.com/) app.
+* [Blorc.PatternFly](https://github.com/WildGums/Blorc.PatternFly) - ![stars](https://img.shields.io/github/stars/WildGums/Blorc.PatternFly?style=flat-square&cacheSeconds=604800) ![last commit](https://img.shields.io/github/last-commit/WildGums/Blorc.PatternFly?style=flat-square&cacheSeconds=86400) Blazor wrappers for [PatternFly](https://www.patternfly.org). To view the latest develop branch in action, visit the [demo](http://blorc-patternfly.wildgums.com/) app.
```
Comment (danroth27) [inline README.md:214]: It looks like http://blorc-patternfly.wildgums.com/ is failing to load: "Failed to find a valid digest in the 'integrity' attribute for resource 'http://blorc-patternfly.wildgums.com/_framework/wasm/dotnet.3.2.0-rc1.20222.2.js' with computed SHA-256 integrity 'yVt8FYsTQDifOGsifIkmEXwe+7ML0jZ1dMi2xluiDXQ='. The resource has been blocked."

Comment (alexfdezsauco) [reply README.md:214]: > It looks like http://blorc-patternfly.wildgums.com/ is failing to load: "Failed to find a valid digest in the 'integrity' attribute for resource 'http://blorc-patternfly.wildgums.com/_framework/wasm/dotnet.3.2.0-rc1.20222.2.js' with computed SHA-256 integrity 'yVt8FYsTQDifOGsifIkmEXwe+7ML0jZ1dMi2xluiDXQ='. The resource has been blocked."

Thanks for reported it.

**Comment by alexfdezsauco** [comment]
@danroth27 in case you still interested to see this https://blorc-patternfly.wildgums.com/

File: README.md (hunk 2 above)
Comment (AdrienTorris) [reply README.md:214]: http://blorc-patternfly.wildgums.com/ just loaded sucessfully.

**Pull request merged**